aeppl/abstract.py

```diff
@@ -15,6 +16,31 @@ class MeasurableVariable(abc.ABC):
 MeasurableVariable.register(RandomVariable)
 
 
+class UnmeasurableMeta(MetaType):
+    def __new__(cls, name, bases, dict):
+        if "id_obj" not in dict:
+            dict["id_obj"] = None
```

Comment:
Here, a check to ensure that id_obj is initialized was added. However, does this make line 41 `id_obj = None` redundant?

Reply:
To my own question: I believe so...